frontend/packages/dev-console/src/plugin.tsx

```diff
@@ -50,6 +51,13 @@ const plugin: Plugin<ConsumedExtensions> = [
       flag: SHOW_PIPELINE,
     },
   },
+  {
+    type: 'FeatureFlag/Model',
```

Comment:
I don't see the flag being used. But it should probably.
We'll need feature checks to enable SBR feature as well as rbac checks